conda-forge.yml

```diff
@@ -1,4 +1,8 @@
-build_platform: {osx_arm64: osx_64}
+build_platform:
+  osx_arm64: osx_64
 conda_forge_output_validation: true
-provider: {win: azure}
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Comment:
> How do you run emulation? 
```suggestion
  linux_ppc64le: azure
```
plus rerender

Reply:
isn't default = azure?

Reply:
Azure is default. My thinking is that a nice test failed.

Reply:
OK, that's a relatively recent change. I had only responded to your "how to run in emulation", sorry.

Reply:
In any case, CI is green now 🥳

Reply:
Gotcha! Thanks for chiming in.

Yeah, I can't merge here. I force pushed to remove the last commit.